serine.galstyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/evn python3
+
+print("Input three different integers:")
+first_num = int(input())
```

Comment:
RW: Get the user input in one line.

Hint: Use string slicing to get the user input numbers.